install/kubernetes/cilium/templates/cilium-agent/validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Capabilities.APIVersions.Has "admissionregistration.k8s.io/v1alpha1/ValidatingAdmissionPolicy" }}
```

Comment:
This is much better, but I think we should also make this dependent on a Helm flag. Many users use `helm template`, in which case Capabilities won't work. And even if they have the capability, they might still want to turn off the validation (it's an alpha feature after all)

I would propose to add a Helm value, e.g. `validaingAdmissionPolicies.enabled` and just have the admission policy created based on that flag.